gatorgrouper/utils/display.py

```diff
@@ -21,10 +21,11 @@ def display_student_groups(student_groups):
 
 def display_welcome_message():
     """ Display a welcome message """
-    print()
-    print("GatorGrouper: Automatically Assign Students to Groups")
-    print("https://github.com/GatorGrouper/gatorgrouper")
-    print()
+    firstline = ""
+    secondline = "GatorGrouper: Automatically Assign Students to Groups"
+    thirdline = "https://github.com/GatorGrouper/gatorgrouper"
+    forthline = ""
```

Comment:
This is ***not*** the same code as before -- was this actually tested by running it at all?

Reply:
This function can be tested by running `pytest`. However, the function itself does not have any output so we will fix that.

Reply:
I have changed it so that the function is printing out the expected output.

Reply:
@Michionlion I reformatted the code in a wired way which I don't know if the program is actually being tested.  It does not make sense to me but it has the correct output and passed `pytest`.

Reply:
By tested I meant the program run at the command line -- what you should have observed is that everything was printed on one line, if at all.